test/ut/retiarii/test_convert_models.py

```diff
@@ -0,0 +1,72 @@
+import os
```

Comment:
Why another file?

Reply:
this file is committed before, it is supposed to test whole models. so I put this test into this file